lib/gax.js

```diff
@@ -90,6 +76,59 @@ function CallSettings(settings) {
 exports.CallSettings = CallSettings;
```

Comment:
Maybe not relevant to this PR, but why do we export `CallSettings`? We decided to make it private in Python.